packages/eslint-plugin/configs/recommended.js

```diff
@@ -21,7 +21,6 @@ const config = {
 	extends: [
 		require.resolve( './recommended-with-formatting.js' ),
 		'plugin:prettier/recommended',
-		'prettier/react',
```

Comment:
Can you explain why this line should be removed?

Reply:
This is the relevant changelog entry: https://github.com/prettier/eslint-config-prettier/blob/5a2f0e24382a8e1322784bc836d47706cccd2f1e/CHANGELOG.md#version-800-2021-02-21

I also documented this in the [commit message](https://github.com/WordPress/gutenberg/pull/36283/commits/60aa8bb40e74f4e91f53746a4d9e1cce2ede293d).